src/index.ts

```diff
@@ -1,65 +1,120 @@
 import { DurableObject } from 'cloudflare:workers';
 
-/**
- * Welcome to Cloudflare Workers! This is your first Durable Objects application.
- *
- * - Run `npm run dev` in your terminal to start a development server
- * - Open a browser tab at http://localhost:8787/ to see your Durable Object in action
- * - Run `npm run deploy` to publish your application
- *
- * Bind resources to your worker in `wrangler.toml`. After adding bindings, a type definition for the
- * `Env` object can be regenerated with `npm run cf-typegen`.
- *
- * Learn more at https://developers.cloudflare.com/durable-objects
- */
-
-/** A Durable Object's behavior is defined in an exported Javascript class */
+// Use npm run cf-typegen to generate the type definitions for the Durable Object
```

Comment:
Is this actually needed? Installing with @cloudflare/workers-types and wrangler as a dependency is usually sufficient.

Reply:
Ah, I didn't know about that. Our docs don't have a lot of examples with `@cloudflare/workers-types`